### PR TITLE
Reorganize course information + Updated schedule

### PR DESCRIPTION
**Pull request by facusapienza21**
I have re-accommodated the order of the information on the course website. I put most of the information in different tabs inside a section called `Overview`, which allow us to split general information, references, syllabus, and reading assignments. 

I have also updated the course schedule based on the current plan for the semester, at least for the next weeks.

### CODE REVIEW

**Comment by facusapienza21** [comment]
@fperez I am still trying to make #3 work, but for the moment you can see the changes in the website from the [JB deployed from my fork](https://facusapienza21.github.io/stat159-site/overview/ra.html).

**Review by fperez — APPROVED** [review]
Excellent, many thanks!!

File: overview/overview.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Statistics 159/259, Spring 2023 Course Summary
+
+**Instructor**: [Fernando Pérez](https://statistics.berkeley.edu/people/fernando-perez), Department of Statistics, UC Berkeley
+
+**Teacher Assistant**: [Facundo Sapienza](https://statistics.berkeley.edu/people/facu-sapienza)
```
Comment (fperez) [inline overview/overview.md]: s/Teacher/Teaching/

File: overview/ra.md [hunk 2]
```diff
@@ -1,39 +1,15 @@
-# Statistics 159/259: Weekly Plan
+# Reading Assignments
 
-**Prof. Pérez and GSI Sapienza , Department of Statistics, UC Berkeley**
+Every week, each student is ask to read and discuss a series of reading assignments. The topics of the reading assignments include reproducibitity in science, open source practices, and applications to climate science, between others.
```
Comment (fperez) [inline overview/ra.md]: s/ask/asked/

Comment (fperez) [reply overview/ra.md]: s/between/among/

File: overview/ra.md [hunk 3]
```diff
@@ -1,39 +1,15 @@
-# Statistics 159/259: Weekly Plan
+# Reading Assignments
 
-**Prof. Pérez and GSI Sapienza , Department of Statistics, UC Berkeley**
+Every week, each student is ask to read and discuss a series of reading assignments. The topics of the reading assignments include reproducibitity in science, open source practices, and applications to climate science, between others. 
 
-Below is our current plan for the course. **This is not a contract:** it is a _plan_, and it may change substantially as the semester unfolds, especially given extra uncertainties due to COVID-19.
+You have to submit a total of two paragraphs per each separate article. The first is a summary paragraph and the second must highlight an important idea of the reading. This last one can be focused in something that catched your attention and you would like to expand a little bit more. You are more than welcome to orient this second paragraph into something related to your own research or area of interest. A good lenght for each one of the paragraphs is between 80 and 120 words
```
Comment (fperez) [inline overview/ra.md]: s/catched/caught/

Comment (fperez) [reply overview/ra.md]: s/lenght/length/

Comment (fperez) [reply overview/ra.md]: Missing end period.

File: overview/ra.md [hunk 4]
```diff
@@ -1,39 +1,15 @@
...
 
-The table of contents on the left pane has links to lectures and labs. The following are live executable links that use the [nbgitpuller](https://jupyterhub.github.io/nbgitpuller) service to give you a current copy from git of the given content, ready to be run in the Spring 2023 Berkeley hub. You can use these links in order to run the content conveniently without manual git work.
+You will submit your reading assignments in bCourses. Reading assignments will be due every Wednesday at 9pm (Exception: the first one will be due _Thursday Jan 26_). No later reading assignments will be accepted unless there is a medical exception. In that case, you will need to submit in Ed a private note to the instructors with a medical proof showing that you are unable to complete the assignment.
```
Comment (fperez) [inline overview/ra.md]: s/no later/no late/

Comment (fperez) [reply overview/ra.md]: s/medical exception/medical situation or other documented major emergency/

Comment (fperez) [reply overview/ra.md]: s/medical proof/supporting evidence/

**Review by fperez — CHANGES_REQUESTED** [review]
Once these small fixes are in, please go ahead and merge. Thanks!!

**Comment by fperez** [comment]
Sorry about a small confusion above - at first I thought the ReadTheDocs config change was a separate PR and meant to approve that change, before realizing that it was part of the main PR. It was just me not reading carefully.

This PR should be merged only after making the few small fixes I highlighted in my review.

Thanks for the great work @facusapienza21!

**Review by facusapienza21 — COMMENTED** [review]
Incorporated gramma check

**Pull request merged**